builder/vsphere/common/step_boot_command.go

```diff
@@ -95,7 +95,7 @@ func (s *StepBootCommand) Run(ctx context.Context, state multistep.StateBag) mul
 			Shift:    keyShift,
 		})
 		if err != nil {
-			return fmt.Errorf("error typing a boot command: %v", err)
+			return fmt.Errorf("error typing a boot command (code, down) `%d, %t`: %w", code, down, err)
```

Comment:
Does `%w` work here? It didn't work for me in this example: https://play.golang.org/p/FkJkN4JFui9

Reply:
%w was added in 1.13 as a way to wrap errors. It works with the fmt.Errorf(), not the fmt.Printf()

Reply:
oh! Sorry, my eyes 👀  did pay attention to the `Errorf` detail!

Reply:
Its not ingrained in my memory yet. I have to look it up each time still.